tests/vmi_multus_test.go

```diff
@@ -578,27 +588,27 @@ var _ = Describe("SRIOV", func() {
 		sriovResourceName = "openshift.io/sriov_net"
 	}
 
-	tests.BeforeAll(func() {
-		tests.BeforeTestCleanup()
-		// Create two sriov networks referring to the same resource name
+	createNetwork := func(networkName string, networkAttachmentDefinition string) {
```

Comment:
You could also have the `namespace` as a parameter of the function. This is optional, you don't need to do it.